src/ui/public/doc_table/doc_table.less

```diff
@@ -74,7 +73,7 @@ doc-table {
  */
 .docTableRowFilterButton {
   appearance: none;
-  background-color: transparent;
+  background-color: @panel-bg;
```

Comment:
I would also add this rule:

```scss
  padding: 0 3px;
```

and remove this rule:

```scss
  & + & {
    margin-left: 5px;
  }
```

This way we can avoid a transparent gap between the buttons.